### PR TITLE
Done fixes for different things to make changes into main template

### PR DESCRIPTION
**Pull request by hardikhexa**
Done fixes for different things to make changes into main template

### CODE REVIEW

File: Application/ZypeAPI/src/main/java/com/zype/fire/api/ZypeSettings.java [hunk 1]
```diff
@@ -8,11 +8,11 @@
 
 public class ZypeSettings {
     // Zype app key
-    public static final String APP_KEY = "C0zChR4FyNVixEOsyJCi1_HLdmklFlj-uUbOAamfEq0f7ML7aJkekOFIbPmbiS9j";
+    public static final String APP_KEY = "_L22X939Nwk6YZde4VvDf8V7iPV15FCYGn-mDI0-1DAznYwYlB1gtsOOZ9lNibCn";
```
Comment (echedev) [inline Application/ZypeAPI/src/main/java/com/zype/fire/api/ZypeSettings.java]: We should not change the app keys for Zype demo build variant

File: Application/app/build.gradle [hunk 2]
```diff
@@ -92,7 +92,7 @@ android {
         zype {
         }
         template {
-            applicationId "<APPLICATION_ID>"
```
Comment (echedev) [inline Application/app/build.gradle]: We should keep the placeholder for 'template' build variant

File: Application/app/src/template/res/values-de/strings.xml [hunk 3]
```diff
@@ -14,7 +14,7 @@ express or implied. See the License for the specific language governing
 permissions and limitations under the License.
 -->
 <resources>
-  <string name="app_name"><APP_NAME></string>
+  <string name="app_name">Test app</string>
```
Comment (echedev) [inline Application/app/src/template/res/values-de/strings.xml]: We should not replace placeholders for 'template' build variant here and in other 'template' files

**Review by echedev — CHANGES_REQUESTED** [review]
There are too many changes in this single commit to figure out the changes related to the issue in APPS-1476 (Inline title style). Please provide smaller commits, or point which changes are specific for each issue.

**Comment by hardikhexa** [comment]
Hello @ech89899 @mattmoore104 , I missed this and now resolved comments to reverted to placeholders.
please check so we can merge this as soon as possible to master.

**Pull request merged**